NewHorizons/Patches/LocatorPatches.cs

```diff
@@ -31,5 +40,89 @@ public static void CloakFieldController_isShipInsideCloak(CloakFieldController _
         {
             __result = __result || Components.CloakSectorController.isShipInside;
         }
+
+        [HarmonyPrefix]
```

Comment:
What are these changes for?

Reply:
Vanilla doesn't register these AstroObjects because Locator.GetAstroObject is never used for them. So I added a way for them to still be registered.

Reply:
It mainly for other modders.

Reply:
I guess theres no harm in it then